repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/ProjectConfigurationRepositoryTest.scala

```diff
@@ -61,14 +82,13 @@ class ProjectConfigurationRepositoryTest extends AsyncWordSpec with Matchers wit
       "return configuration by project id" in {
         when(documentRepository.getByParam("projectId", projectId.value))
           .thenReturn(Future.successful(List(configuration)))
-        configurationRepository.getById(projectId).map(_ shouldBe Some(configuration))
+        configurationRepository.getAllByProjectId(projectId).map(_.headOption shouldBe Some(configuration))
```

Comment:
You don't need `headOption` here.
```suggestion
        configurationRepository.getAllByProjectId(projectId).map(_ shouldBe List(configuration))
```
Same for test below

Reply:
Yep. Fixed.